frontend/src/app/projects/create-project/create-project.component.html

```diff
@@ -6,64 +6,87 @@
+                </mat-error>
+              </mat-form-field>
+            </fieldset>
+            <fieldset>
+              <mat-form-field appearance="fill">
+                <mat-label>Description</mat-label>
+                <textarea matInput formControlName="description"></textarea>
+              </mat-form-field>
+            </fieldset>
+            <div class="flex-space-between">
+              <a routerLink="/projects" mat-button matStepperPrevious>
+                <app-mat-icon position="left" usage="text"
+                  >arrow_back</app-mat-icon
+                >
+                Cancel</a
+              >
+              <button
+                mat-flat-button
+                color="primary"
+                [disabled]="!form.valid"
+                (click)="createProject(stepper)"
+              >
+                Create project
+                <app-mat-icon position="right" usage="text"
+                  >arrow_forward</app-mat-icon
+                >
+              </button>
+            </div>
+          </form>
+        </mat-card>
+      </div>
     </mat-step>
 
     <mat-step label="Add team members">
-      <mat-card class="section-card" *ngIf="projectService.project">
-        <h2>Your role will be project lead.</h2>
-        <app-project-user-settings
-          [repository]="projectService.project"
-        ></app-project-user-settings>
-          <button mat-flat-button color="primary" (click)="stepper.next()">
-            Add models
-          </button>
-          <div class="field-separator"></div>
-        <button mat-flat-button color="primary" (click)="finish()">
-          Finish project creation
-        </button>
-      </mat-card>
+      <div *ngIf="projectService.project" class="flex-center">
+        <div>
+          <app-project-user-settings
+            [project]="projectService.project"
+          ></app-project-user-settings>
+          <mat-card class="mat-card-default">
+            <div class="flex-space-between">
+              <button mat-flat-button color="primary" (click)="stepper.next()">
+                Skip
+                <app-mat-icon usage="button" class="mat-icon-position right"
+                  >arrow_forward</app-mat-icon
+                >
+              </button>
+            </div>
+          </mat-card>
+        </div>
+      </div>
     </mat-step>
     <mat-step label="Add models">
-      <app-create-model
-        *ngIf="!_reload"
-        [as_stepper]="true"
-        (complete)="onComplete($event)"
-        #model_creator
-      ></app-create-model>
+      <app-create-model #model_creator></app-create-model>
+      <div class="flex-center">
+        <mat-card class="section-card">
+          <div class="flex-center">
+            <a
+              mat-flat-button
+              [routerLink]="['/project', projectService.project]"
+            >
+              Finish project creation
+              <app-mat-icon class="mat-icon-position right">check</app-mat-icon>
```

Comment:
The `[inline]` property of th`e `mat-icon should be used there. Since there isn’t any size concerns there (the icon size it the police size), `<mat-icon [inline]>` can be used

Reply:
As discussed, the `[inline]` modifies the size of the icon. It has been fixed with replacing it back to `<mat-icon>`.